cli/src/config/colors.toml

```diff
@@ -23,6 +23,7 @@
 
 "email" = "yellow"
 "username" = "yellow"
+"log author name" = "yellow"
```

Comment:
I'm not pretty sure, but maybe it's okay to colorize `author` and `committer`, and override the `timestamp` field?
```toml
"author" = "yellow"
"committer" = "yellow"
"timestamp" = "cyan"
```

This means author/committer names in `jj show` are colorized, but I think it's more consistent.

Reply:
Looks much better ❤️. I'll give that a shot.